src/styles/blocks/card.scss

```
@@ -0,0 +1,104 @@
+.card {
+  padding: 33px 17px 17px;
```

```suggestion
  padding: 32px 16px 16px;
```
with setting box-sizing fix paddings can help to fix failed tests.